Pomodoro/app/src/main/java/no/uia/ikt205/pomodoro/MainActivity.kt

```diff
@@ -1,99 +1,161 @@
 package no.uia.ikt205.pomodoro
 
+import android.app.assist.AssistContent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.os.CountDownTimer
+import android.text.Editable
+import android.text.TextWatcher
 import android.view.View
-import android.widget.Button
-import android.widget.TextView
-import android.widget.Toast
+import android.widget.*
 import no.uia.ikt205.pomodoro.util.millisecondsToDescriptiveTime
 
-class MainActivity : AppCompatActivity() {
 
-    var isRunning: Boolean = false;
 
+class MainActivity : AppCompatActivity() {
+
+    lateinit var pauseTimer: CountDownTimer
     lateinit var timer: CountDownTimer
```

Comment:
Kunne brukt tydeligere navn her for å spesifisere hvilken tid slik du gjorde med "pauseTimer"